src/main/kotlin/com/infusionvlc/somniumserver/dreams/models/Dream.kt

```diff
@@ -1,12 +1,12 @@
 package com.infusionvlc.somniumserver.dreams.models
 
 data class Dream(
-  val id: Long,
-  val title: String,
-  val description: String,
-  val userId: Long,
-  val public: Boolean,
-  val creationDate: Long,
-  val updateDate: Long,
-  val dreamtDate: Long
+  val id: Long = 0,
```

Comment:
Default variables are a mistake, you can create something it couldn't exist like a dream without ID, or dream without title.